concent_api/api-integration-force-payment.py

```diff
@@ -49,25 +49,23 @@ def test_case_2d_send_correct_force_payment(cluster_consts, cluster_url, test_id
     correct_force_payment = force_payment(
         subtask_results_accepted_list=[
             subtask_results_accepted(
-                payment_ts=current_time - cluster_consts.payment_due_time - 10,
+                payment_ts=current_time - cluster_consts.payment_due_time - 30,
```

Comment:
Please create utility function, as this calculation is being repeated several times.

Reply:
Done